kitsune/users/tests/test_models.py

```diff
@@ -28,3 +28,53 @@ def test_default_values(self):
         for setting in keys:
             field = SettingsForm.base_fields[setting]
             eq_(field.initial, Setting.get_for_user(self.u, setting))
+
+
+class AccountEventTests(TestCase):
+
+    def test_process_delete_user(self):
```

Comment:
Let's not have these events through cron. We can `apply` the task through celery as they come in. After that we can start exploring solutions (check the celery result, cron tasks) for failed/errored tasks.